compiler/ilgen/MethodBuilder.hpp

```diff
@@ -73,7 +74,7 @@ class MethodBuilder : public TR::IlBuilder
 
    TR::TypeDictionary *typeDictionary()                      { return _types; }
 
-   const char *getDefiningFile()                             { return _definingFile; }
+   std::string getDefiningFile()                             { return _definingFile; }
```

Comment:
I had (mistakenly) included the type change for `_definingFile` along with the next commit. I have rebased my commits to include it as a part of this now.